src/lumigo_tracer/utils.py

```diff
@@ -220,6 +226,7 @@ def report_json(region: Union[None, str], msgs: List[dict]) -> int:
             get_logger().info(f"successful reporting, code: {getattr(response, 'code', 'unknown')}")
         except Exception as e:
             get_logger().exception(f"Could not report json to {host}", exc_info=e)
+            edge_connection = establish_connection(host)
```

Comment:
Would have been nice to add to the log message (upper line), the word retrying.
It's not a full fix, right? this request is lost, the next one hopefully will succeed.

Reply:
From what I saw even "clean" lambdas fail, so not sure how recreating the connection will help

Reply:
My guess is that it happens in warm lambdas, that loses their connection between the invocations somehow...
It is indeed not a full fix because I need to validate it, but that's absolutely a solid start

Reply:
fixed log message 👍